src/stack.js

```diff
@@ -1,5 +1,21 @@
-class Stack {
+// Should have the methods: `add`, `remove`, and a getter for the property `size`
+ // `add` should accept a value and place it on top of the stack.
+ // `remove` should remove and return the top value off of the stack.
+ // `size` should return how many items are on the stack.
 
+class Stack {
+  constructor() {
+    this.stack = [];
+  }
+  get size() {
+    return this.stack.length;
+  }
+  add(val) {
+    this.stack.push(val);
+  }
+  remove(val) {
```

Comment:
The `pop()` function doesn't accept any input. It just pops off the element that was last added to it. So there's no need to have the remove function accept a value.